From 236ebb968a298fa5d461e734559ad8a13b667eb6 Mon Sep 17 00:00:00 2001
From: Jon Mason <jon.mason@arm.com>
Date: Wed, 24 Jan 2024 11:35:50 -0500
Subject: [PATCH] xtest: stats: remove unneeded stat.h include

Hack to work around musl compile error:

| In file included from optee-test/4.1.0/recipe-sysroot/usr/include/sys/stat.h:23,
|                  from optee-test/4.1.0/git/host/xtest/stats.c:17:
| optee-test/4.1.0/recipe-sysroot/usr/include/bits/stat.h:17:26: error: expected identifier or '(' before '[' token
|    17 |         unsigned __unused[2];
|       |                          ^

stat.h is not needed, since it is not being used in this file.  So
removing it.

Upstream-Status: Inappropriate [https://github.com/OP-TEE/optee_test/issues/722]
Signed-off-by: Jon Mason <jon.mason@arm.com>
---
 host/xtest/stats.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/host/xtest/stats.c b/host/xtest/stats.c
index fb16d55586da..05aa3adac611 100644
--- a/host/xtest/stats.c
+++ b/host/xtest/stats.c
@@ -14,7 +14,6 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-#include <sys/stat.h>
 #include <sys/types.h>
 #include <tee_client_api.h>
 #include <unistd.h>
